src/guide/testing.md

```diff
@@ -1,165 +1,165 @@
-# Testing
+# Pruebas
 
-## Introduction
+## Introducción
 
-When it comes to building reliable applications, tests can play a critical role in an individual or team's ability to build new features, refactor code, fix bugs, etc. While there are many schools of thought with testing, there are three categories often discussed in the context of web applications:
+Cuando se refiere a desarrollar aplicaciones de confianza, las pruebas pueden jugar un papel fundamental en la habilidad un individuo o equipo para desarrollar nuevas características, refactorizar código, mitigar errores, etc. Aunque existen muchas escuelas de pensamiento con las pruebas, hay tres categorías a menudo discutidas en el contexto de aplicaciones web:
```

Comment:
What do you think about this? @lagp0310 

```suggestion
Cuando se trata de desarrollar aplicaciones de confianza, las pruebas pueden desempeñar un papel fundamental en la capacidad de un individuo o un equipo, para desarrollar nuevas características, refactorizar código, mitigar errores, etc. Aunque existen muchas escuelas de pensamiento con las pruebas, hay tres categorías a menudo discutidas en el contexto de aplicaciones web:
```

Reply:
Much better and more formal way to translate it! :smile: